src/Control/Lens/At.hs

```diff
@@ -469,10 +469,14 @@ instance Ord k => At (Map k a) where
   {-# INLINE at #-}
 
 instance (Eq k, Hashable k) => At (HashMap k a) where
+#if MIN_VERSION_unordered_containers(0,2,10)
```

Comment:
Just bump the lower bound on `unordered-containers`.

Reply:
For reference, https://matrix.hackage.haskell.org/#/package/unordered-containers `-0.2.10.0` supports GHC-7.8, which is the oldes GHC next `lens` release would support. So there is little reason to extend support to older `unordered-containers`.